install-system.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+
+if [[ $# -gt 1 ]]; then
+    echo "Usage: install.sh [--flash | --no-flash]" >&2
+    exit 1
+fi
+
+if [[ $# -gt 0 ]]; then
+    case "$1" in
+        "--flash")
+            flash=true
+            ;;
+        "--no-flash")
+            flash=false
+            ;;
+        *)
+            echo "Usage: install.sh [--flash | --no-flash]" >&2
+            exit 1
+            ;;
+    esac
+else
+    echo "Would you like to install Adobe Flash Player? (Only required for crawls with Flash) [y,N]"
+    read -s -n 1 response
+    if [[ $response = "" ]] || [ $response == 'n' ] || [ $response == 'N' ]; then
+        flash=false
+        echo Not installing Adobe Flash Plugin
+    elif [ $response == 'y' ] || [ $response == 'Y' ]; then
+        flash=true
+        echo Installing Adobe Flash Plugin
+    else
+        echo Unrecognized response, exiting
+        exit 1
+    fi
+fi
+
+if [ "$flash" = true ]; then
+    sudo sh -c 'echo "deb http://archive.canonical.com/ubuntu/ trusty partner" >> /etc/apt/sources.list.d/canonical_partner.list'
```

Comment:
Let's update this to `sudo add-apt-repository "deb http://archive.canonical.com/ $(lsb_release -sc) partner"` so we use the correct archive for whatever version of the OS the user is using.

Reply:
Done